README.md

```diff
@@ -221,6 +221,7 @@ Table of Contents
   * https://www.asp.net/ajax/cdn - The Microsoft Ajax Content Delivery Network (CDN) hosts popular third party JavaScript libraries such as jQuery and enables you to easily add them to your Web application
   * https://toranproxy.com/ - Proxy for Packagist and GitHub. Never fail CD. Free for personal use, 1 developer, no support.
   * http://rawgit.com - free limited traffic, serves raw files directly from GitHub with proper Content-Type headers.
+  * https://tech.yandex.ru/jslibs/ - popular open-source JavaScript frameworks and libraries
```

Comment:
I think a russian based CDN for JS libs is probably a bit niche for this list, it's mainly aimed at *aaS for devops and infradev types, so I dont think this one fits the bill sorry